### PR TITLE
LPS-151390 Add a missing `Liferay.PropsValues` type definition

### PR DESCRIPTION
**Pull request by kresimir-coko**
Fixes a concern raised in [this issue](https://issues.liferay.com/browse/PTR-2989).

### CODE REVIEW

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
> fatal: Not a valid branch point: '4389d11f0d64c60b9ef6a2cc808cedf46d330f0d'.

Go home CI, you're drunk

**Comment by diegonvs** [comment]
ci:test:relevant

**Comment by kresimir-coko** [comment]
ci:test:relevant

File: modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts [hunk 1]
```diff
@@ -487,6 +487,10 @@ declare module Liferay {
 		export function getLanguageId(): Language.Locale;
 	}
 
+	namespace PropsValues {
```
Comment (bryceosterhaus) [inline modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts:490]: Can we also sort this file alphabetically? I suspect Brian will bring that up sooner or later, you can just add it as an additional commit.

Comment (kresimir-coko) [reply modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/liferay.d.ts]: Done in https://github.com/liferay-frontend/liferay-portal/pull/2115/commits/7ef3d43c2c1410acec521b9e3f2b7bd047db824e

**Comment by kresimir-coko** [comment]
ci:test:sf

**Comment by diegonvs** [comment]
ci:test:relevant

**Review by bryceosterhaus — COMMENTED** [review]
ci:forward

**Comment by bryceosterhaus** [comment]
ci:forward

**Pull request closed without merging**